### ISSUE TITLE
Font Color - Breaks everything

### ISSUE THREAD

**Issue by ubo32** [HappenApps/Quiver#902]
Trying to change the color of the font is very hit-or-miss. When selecting an entire line, and choose the color, nothing happens. I have to omit one character, choose the color, then select the omitted character and then change its color. 

Selecting multiple lines and changing the font colors does nothing. Sometimes it even breaks the formatting. It makes the app unusable. 

This is all in a Text cell. 

Any suggestions?

**Comment by justnealpatel** [comment]
I agree. This has been an issue for me. I found it a lot easier to just style the text through CSS so the text stands out more, assuming that you have specific types of text (i.e. bold text or inline code) that you're trying to change. This bug needs to be fixed though.

**Comment by BenceSzalai** [comment]
This is still an issue in v3.2.7 (201).
An alternative solution could be to allow a raw view for Text cells, where the (I assume HTML) markup can be edited/fixed.
